src/webRTCEndpoint.ts

```diff
@@ -406,6 +406,10 @@ export class WebRTCEndpoint extends (EventEmitter as new () => TypedEmitter<
     }
   };
 
+  get tracks(): Readonly<Record<string, TrackContext>> {
+    return Object.fromEntries(this.trackIdToTrack.entries());
+  }
```

Comment:
Would be nice to add some docs or rename `tracks` to `remoteTracks` . I would favor the second option?